sdk/storage/storage-blob-changefeed/rollup.base.config.js

```diff
@@ -55,6 +56,15 @@ export function nodeConfig(test = false) {
       cjs()
     ],
     onwarn(warning, warn) {
+      if (
+        warning.code === "CIRCULAR_DEPENDENCY" &&
+        warning.importer.indexOf(path.normalize("node_modules/@opentelemetry/api")) >= 0
+      ) {
+        // opentelemetry contains circular references but it doesn't cause issues.
+        // Tracked in https://github.com/open-telemetry/opentelemetry-js-api/issues/87
```

Comment:
Pretty self explanatory, we just need to centralize this at some point (we have issues tracking that work)